Model/Api/Customer.php

```diff
@@ -4,16 +4,30 @@
 
 use Exception;
 use OmiseCustomer;
+use Omise\Payment\Model\Config\Config;
 
 class Customer extends BaseObject
 {
+    /**
+     * Injecting dependencies
+     * @param \Omise\Payment\Model\Config\Config $config
+     * @param \Magento\Store\Model\StoreManagerInterface $storeManager
+     */
+    public function __construct(Config $config)
+    {
+        $this->config = $config;
+    }
+
     /**
      * @param  string $id
      *
      * @return Omise\Payment\Model\Api\Error|self
      */
     public function find($id)
     {
+        if (!$this->config->canInitialize()) {
+            return;
+        }
```

Comment:
This prevents our plugin from triggering when no keys are configured. This is related to this PR(https://github.com/omise/omise-magento/pull/339). Just moved the code to the `Config.php` from `Capabilities.php` so that it can be used in other API classes when necessary.